### ISSUE TITLE
Cannot run the run command

### ISSUE THREAD

**Issue by HosseinMA96** [jvlmdr/flax-cifar#1]
Hi.

I have installed all dependencies, but I cannot run the run command.

`python main.py --dataset_root=path/to/cifar10 --config=configs/default.py --config.model.arch=resnet_v1_18
`

the error is:
`Traceback (most recent call last):
  File "/home/hossein/tmp/scalable influence/jax-influence/CIFAR/flax-cifar-main/main.py", line 241, in <module>
    app.run(main)
  File "/home/hossein/anaconda3/envs/virginGPU/lib/python3.9/site-packages/absl/app.py", line 300, in run
    args = _run_init(
  File "/home/hossein/anaconda3/envs/virginGPU/lib/python3.9/site-packages/absl/app.py", line 369, in _run_init
    args = _register_and_parse_flags_with_usage(
  File "/home/hossein/anaconda3/envs/virginGPU/lib/python3.9/site-packages/absl/app.py", line 216, in _register_and_parse_flags_with_usage
    args_to_main = flags_parser(original_argv)
  File "/home/hossein/anaconda3/envs/virginGPU/lib/python3.9/site-packages/absl/app.py", line 156, in parse_flags_with_usage
    return FLAGS(args)
  File "/home/hossein/anaconda3/envs/virginGPU/lib/python3.9/site-packages/absl/flags/_flagvalues.py", line 644, in __call__
    unknown_flags, unparsed_args = self._parse_args(args, known_only)
  File "/home/hossein/anaconda3/envs/virginGPU/lib/python3.9/site-packages/absl/flags/_flagvalues.py", line 793, in _parse_args
    flag.parse(value)
  File "/home/hossein/anaconda3/envs/virginGPU/lib/python3.9/site-packages/absl/flags/_flag.py", line 168, in parse
    self.value = self._parse(argument)
  File "/home/hossein/anaconda3/envs/virginGPU/lib/python3.9/site-packages/ml_collections/config_flags/config_flags.py", line 625, in _parse
    overrides_types = GetTypes(overrides, config)
  File "/home/hossein/anaconda3/envs/virginGPU/lib/python3.9/site-packages/ml_collections/config_flags/config_flags.py", line 912, in GetTypes
    return [GetType(path, config) for path in paths]
  File "/home/hossein/anaconda3/envs/virginGPU/lib/python3.9/site-packages/ml_collections/config_flags/config_flags.py", line 912, in <listcomp>
    return [GetType(path, config) for path in paths]
  File "/home/hossein/anaconda3/envs/virginGPU/lib/python3.9/site-packages/ml_collections/config_flags/config_flags.py", line 893, in GetType
    current = _TakeStep(current, step)
  File "/home/hossein/anaconda3/envs/virginGPU/lib/python3.9/site-packages/ml_collections/config_flags/config_flags.py", line 877, in _TakeStep
    return _AccessConfig(current, field, indices)
  File "/home/hossein/anaconda3/envs/virginGPU/lib/python3.9/site-packages/ml_collections/config_flags/config_flags.py", line 853, in _AccessConfig
    current = current[field]
KeyError: 'model'
`

Apparently the current dictionary does not have `model` key in it in the absl.apply . I inspected the current dictionary and it was this:
 {'arch': 'wrn28_2', 'train': {'base_learning_rate': 0.1, 'num_epochs': 50, 'batch_size': 128, 'weight_decay': 0.0003, 'weight_decay_vars': 'all'}}

I am also not sure if there is a pre trained model or if I am supposed to train.

Thanks.

**Comment by jvlmdr** [comment]
Hey! Thanks for reporting this! It looks like the command in the readme is incorrect. Try using `--config.arch` instead of `--config.model.arch`:
```bash
python main.py --dataset_root=path/to/cifar10 --config=configs/default.py --config.arch=resnet_v1_18
```

**Comment by jvlmdr** [comment]
Also, to respond to your question, there are currently no pre-trained models; you need to train from scratch. (It doesn't take too long to train on CIFAR with one GPU.)

**Comment by HosseinMA96** [comment]
Thanks for the reply. Do you also know how to save the model?

**Comment by jvlmdr** [comment]
Check out the flax docs (https://flax.readthedocs.io/en/latest/guides/use_checkpointing.html). Feel free to implement this and make a pull request. Otherwise, I'll get around to it when I can.

I'm closing this specific issue as fixed by 6a0e0ba. Feel free to open a new issue for checkpointing / saved models!
